rpcs3/Emu/RSX/Overlays/overlay_message_dialog.cpp

```diff
@@ -256,12 +256,18 @@ namespace rsx
 						}
 						else
 						{
-							while (!exit)
+							while (!exit && thread_ctrl::state() == thread_state::created)
 							{
 								refresh();
 
 								// Only update the screen at about 60fps since updating it everytime slows down the process
 								std::this_thread::sleep_for(16ms);
```

Comment:
maybe use thread_ctrl::wait() here.

Reply:
Well, I'm not touching this parts of code.